### DISCUSSION TITLE
Google cloud charges with "SPENDING_BASED_DISCOUNT"

### DISCUSSION THREAD

**Discussion by yair-ros** [NightscoutFoundation/xDrip#2789] [Q&A] [answered]
Hi everyone, 
I read the documentaion about "[Invoice for egress to China](https://navid200.github.io/xDrip/docs/Nightscout/Crawlers.html)" and I also followed the "[Nightscout on Google Cloud Cost Table](https://navid200.github.io/xDrip/docs/Nightscout/CostTable.html)" and I can see on my cost table that I have some "SPENDING_BASED_DISCOUNT" discounts.

I afraid that I will need to pay on that in the future.
Currently it is a small amount, but I can't understand why I have been charged on it and how can I prevent this in the future.
Attached is a screenshot of all my charges and discounts.
![image](https://user-images.githubusercontent.com/7479809/233058059-46bdf2ac-2e7a-4348-8163-a6480dd98418.png)

**Dose anyone familiar with that?**

Thanks.

**Comment by Navid200** [comment, marked as answer]
Not all the discounts disappear after you trial period ends.
I have been using it since November and I still have discounts.

The way to look at this, if you are still in the trial period, is to look at billing.  It will show you have much credit you still have and what the original credit was.  From that, you can see how much you would be charged over time if you did not have the free trial.

Can you look at that page and post a screenshot?

**Reply by yair-ros** [reply]
Hi @Navid200 
Thanks for the very fast response!
Actually, after I read your answer, I checked when I have been created my Nightscout google server instance and I realized that I have been using it since October and I still have discounts...

So, seems that everything is ok for now.

Thanks again.
